pkg/sentry/fs/proc/sys_net.go

```diff
@@ -280,11 +280,120 @@ func (p *proc) newSysNetCore(ctx context.Context, msrc *fs.MountSource, s inet.S
 	return newProcInode(ctx, d, msrc, fs.SpecialDirectory, nil)
 }
 
+// ipForwarding implements fs.InodeOperations.
+// ipForwarding is used to enable/disable packet forwarding of netstack
+// and cache user input to "/proc/sys/net/ipv4/ip_forward".
+//
+// +stateify savable
+type ipForwarding struct {
+	stack inet.Stack `state:"wait"`
+	fsutil.SimpleFileInode
+
+	// mu protects enabled
+	mu sync.Mutex
```

Comment:
You need to mark this as `state:"nosave"`.

Reply:
fixed